src/dss/utils.py

```diff
@@ -81,3 +113,79 @@ def get_lightkube_client(kubeconfig: Optional[str] = None):
     kubeconfig = KubeConfig.from_file(kubeconfig)
     lightkube_client = Client(config=kubeconfig)
     return lightkube_client
+
+
+def get_mlflow_tracking_uri() -> str:
+    return f"http://{MLFLOW_DEPLOYMENT_NAME}.{DSS_NAMESPACE}.svc.cluster.local:5000"
+
+
+def get_notebook_url(name: str, namespace: str, lightkube_client: Client) -> str:
```

Comment:
Can we rename this to get_service_url ... we can reuse it also for other services not just notebooks

Reply:
Updated